charts/gateway/README.md

```diff
@@ -61,11 +61,58 @@ Helm Version    Supported Kubernetes Versions
 * [Upgrade the Chart](#upgrading-the-chart)
 * [Uninstall the Chart](#uninstalling-the-chart)
 
+## Additional Guides
+* [Service Configuration](#port-configuration)
+* [Gateway Application Ports](#gateway-application-ports)
+* [Ingress Configuration](#ingress-configuration)
+* [PM Tagger Configuration](#pm-tagger-configuration)
+* [Redis Configuration](#redis-configuration)
+* [OpenTelemetry Configuration](#opentelemetry-configuration)
+* [OTK Install or Upgrade](#otk-install-or-upgrade)
+* [Database Configuration](#database-configuration)
+* [Cluster-Wide Properties](#cluster-wide-properties)
+* [Java Args](#java-args)
+* [System Properties](#system-properties)
+* [Gateway Bundles](#bundle-configuration)
+* [Bootstrap Script](#bootstrap-script)
+* [Custom Health Checks](#custom-health-checks)
+* [Custom Configuration Files](#custom-configuration-files)
+* [Logs & Audit Configuration](#logs--audit-configuration)
+* [Graceful Termination](#graceful-termination)
+* [Autoscaling](#autoscaling)
+* [Pod Disruption Budgets](#pod-disruption-budgets)
+* [RBAC Parameters](#rbac-parameters)
+* [Service Metrics Demo](#service-metrics-demo)
+* [SubChart Configuration](#subchart-configuration)
+
+# Java 17
+The Layer7 API Gateway is now running with Java 17 with the release of v11.1.00.
+
+If you use Policy Manager, you will need to update to v11.1.00.
+
 # Java 11
```

Comment:
Do we still need to reference GW 10.1? 10.1 will be EOL soon.

Reply:
Good point, I'll save that for a PR nearer EOL.